### DISCUSSION TITLE
How to set Input and Output Dates to ISO standards

### DISCUSSION THREAD

**Discussion by StevenTCramer** [statiqdev/Discussions#105] [Q&A]
I have been hoping the world will adopt the ISO date standard .. Most Significant to least.  2021-11-26

Where can I configure this in Statiq?

**Comment by daveaglick** [comment]
There are actually two settings specifically for this purpose (though it's up to themes and layouts to make use of them):
- `DateTimeInputCulture`: Indicates the culture to use for reading and interpreting dates as input.
- `DateTimeDisplayCulture`: Indicates the culture to use for displaying dates in output.

Let me know if that does what you're looking for or if we need to go deeper.
